src/main/java/hudson/plugins/git/util/GitUtils.java

```diff
@@ -53,6 +53,35 @@ public static Node workspaceToNode(FilePath workspace) { // TODO https://trello.
     }
 
     /**
+     * Call close method on walk using reflection.  JGit 3 uses
```

Comment:
I think you're saying that it would be better to insert an _release method into GitUtils which then either calls the release() method or the close(0 method.  That seems like a much better approach.